src/lib/components/state/AccountDetailEmptyState.tsx

```diff
@@ -0,0 +1,35 @@
+import type { FlexProps, TextProps } from "@chakra-ui/react";
+import { Flex, Text } from "@chakra-ui/react";
+
+export interface AccountDetailEmptyStateProps {
+  message: string;
+  textVariant?: TextProps["variant"];
+  pt?: FlexProps["pt"];
+}
+
+export const AccountDetailEmptyState = ({
```

Comment:
move to `account-details/components/` and rename both filename and the component name to `AccountDetailsEmptyState`